packages/office-addin-project/README.md

```diff
@@ -6,13 +6,15 @@ For more information, see the [documentation](
 https://docs.microsoft.com/en-us/office/dev/add-ins/develop/add-in-manifests).
 
 ## Command-Line Interface
+
 * [convert](#info)
 
 #
 
 ### convert
 
 Converts the Office Addin Code from xml to json based manifest.
+The outputed conversion is avaible at `./manifest`.
```

Comment:
This is converting the project and not just the manifest so this could also be misleading.

Reply:
Agree, the documentation could have the details explaining the resulting project structure (like where is the manifest).